README.md

```diff
@@ -17,51 +22,61 @@ NOTE: no database setup here. Backend uses list-based live data
 
```

Comment:
On step 6 it sounds like Windows might replace "bin" with "scripts", we could add a "If you are on windows, do this, if on Mac do this" line